capitalize_decorator.rb

```diff
@@ -0,0 +1,8 @@
+# capitalize_decorator.rb
+require_relative 'decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    super.capitalize
+  end
```

Comment:
- [ ] Please make sure to call the `correct_name` method from the parent class and capitalize the nameable variable.
   - You can consider using this code:
   
   
 ```ruby
def correct_name
      @nameable.correct_name.capitalize
    end
```